cloudbuild_public.yaml

```diff
@@ -0,0 +1,70 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: 'bash'
+  args: [
+    '-c',
+    'docker pull us.gcr.io/$PROJECT_ID/$REPO_NAME:nightly || true',
+  ]
+- name: 'gcr.io/cloud-builders/docker'
+  args: 
+    [
+      'build',
+      '-t', 'us.gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA',
+      '-t', 'us.gcr.io/$PROJECT_ID/$REPO_NAME:nightly',
+      '--cache-from', 'us.gcr.io/$PROJECT_ID/$REPO_NAME:nightly',
+      '--cache-from', 'us.gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA',
+      '.'
+    ]
+# CDC tests
+- name: 'gcr.io/$PROJECT_ID/docker-compose'
+  args:
+    - '-f'
+    - 'docker-compose.gcb.yml'
+    - run
+    - '--rm'
+    - cdc-test
+# Clean up after tests
+- name: 'gcr.io/$PROJECT_ID/docker-compose'
+  args:
+    - '-f'
+    - 'docker-compose.gcb.yml'
+    - down
+    - '--rmi'
+    - 'local'
+    - '-v'
+    - '--remove-orphans'
+# Only tag "nightly" when on master
+- name: 'gcr.io/cloud-builders/docker'
+  secretEnv: ['DOCKER_PASSWORD']
+  entrypoint: 'bash'
+  args:
+  - '-e'
+  - '-c'
+  - |
+    # Only push to Docker Hub from master
+    [ "$BRANCH_NAME" != "master" ] && exit 0
+    docker push us.gcr.io/$PROJECT_ID/$REPO_NAME:nightly
+    echo "$$DOCKER_PASSWORD" | docker login --username=sentrybuilder --password-stdin
+    docker tag us.gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA $PROJECT_ID/$REPO_NAME:$SHORT_SHA
+    docker push $PROJECT_ID/$REPO_NAME:$SHORT_SHA
+    docker tag us.gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA $PROJECT_ID/$REPO_NAME:$COMMIT_SHA
+    docker push $PROJECT_ID/$REPO_NAME:$COMMIT_SHA
+    docker tag us.gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA $PROJECT_ID/$REPO_NAME:nightly
+    docker push $PROJECT_ID/$REPO_NAME:nightly
+images:
+  [
+    "us.gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA",
+  ]
+timeout: 3600s
+options:
+  # Run on bigger machines
```

Comment:
Do we need this for CDC? Mind you, these machines are twice as expensive. Also, this comment reminds me of:

![this is bridge](https://user-images.githubusercontent.com/126780/113401320-dd836a80-93ab-11eb-8297-a054da52b3f7.png)

Reply:
Sorry, copy paste from symbolicator I forgot to remove.

Reply:
Removed